lib/cassandra_migrations/cql-rb-wrapper.rb

```diff
@@ -48,6 +47,7 @@ def use(keyspace)
     if @sessions[keyspace]
       @session = @sessions[keyspace]
     else
+      Rails.logger.try(:info, "Creating Cassandra session: #{keyspace.inspect}")
```

Comment:
@hsgubert for me, seeing the additional logging of cluster connection, session creation and disconnection is more beneficial, whether I'm using spring, passenger, or puma, or anything else. These, in the context of other logging I might already have (like when a process forks, etc), are what I think I would expect. 

```ruby
=> Booting Thin
=> Rails 4.2.4 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
I, [2015-11-19T14:29:06.310815 #7758]  INFO -- : Connecting to Cassandra cluster: {:hosts=>["127.0.0.1"], :port=>9042}
I, [2015-11-19T14:29:06.414818 #7758]  INFO -- : Creating Cassandra session: [no keyspace]
I, [2015-11-19T14:29:06.417689 #7758]  INFO -- : Creating Cassandra session: some_cassandra_app_development"
=> Rack::Timeout set to 1800 in development
>> Thin web server (v1.5.1 codename Straight Razor)
>> Maximum connections set to 1024
>> Listening on localhost:3000, CTRL+C to stop
```

Happy to hear your thoughts. If you want additional lines for the passenger fork, happy to add them back.